tex/gregoriotex-syllable.tex

```diff
@@ -556,7 +556,7 @@
   % took me years to find it!
   \gre@debugmsg{bolshift}{Current glyph: \the\gre@attr@glyph@id}%
   \ifgre@bolshiftsenabled%
-    \gre@calculate@bolshift{\gre@dimen@begindifference}%
+    \gre@calculate@bolshift{\gre@dimen@begindifference}{\gre@firstisalteration}% TODO: compute firstisalteration
```

Comment:
Indeed (I admit I tend to prefer to make code as explicit as possible, having many globals make it more difficult to understand (hence the additional argument to calculate@bolshift)